src/include/connection.h

```diff
@@ -92,6 +103,9 @@ struct __wt_background_compact {
 
     /* List of files to track compaction statistics across background server iterations. */
     TAILQ_HEAD(__wt_bg_compacthash, __wt_background_compact_stat) * compacthash;
+    /* List of files excluded from background compaction. */
+    TAILQ_HEAD(__wt_bg_compactexcludeentryhash, __wt_background_compact_exclude_entry) *
+      compactexcludeentryhash;
```

Comment:
Not a big fan of this name, anyone has better suggestions?

Reply:
Since this is already part of the background_compact structure must we have the term `compact` in the name? Could we make this `excludehash` and rename the list above to `stathash` or something similar?

Reply:
I second that. Also while reading the code I was feeling we could abbreviate "__wt_background_compact_..." to something else, and you have already done that here, why not throughout the code call "__wt_bg_compact_..."

Reply:
We had this discussion with @agorrod and we would like to avoid abbreviations.

Reply:
See 434d9240edd6785d13ac2bad8187db7b03de9bef for the `stathash` related changes and cc52c025926072ed826202d782fc9641f879cd2a for the `excludehash` related changes.